src/pages/ProgramContentPage/ProgramContentBlock.tsx

```diff
@@ -226,121 +226,50 @@ const ProgramContentBlock: React.VFC<{
-          </StyledBackgroundModeDescriptionBlock>
-        )}
-      {!!isEquityProgramContent && programContent.contentType === 'video' ? (
-        (!isBackgroundMode || programContent.videos[0]?.data?.source === 'youtube') &&
-        ((!!isEquityProgramContent && moment().isAfter(moment(programContent.publishedAt))) ||
-          currentUserRole === 'app-owner') && (
-          <ProgramContentPlayer
-            programContentId={programContentId}
-            nextProgramContent={nextProgramContent}
-            onVideoEvent={e => {
-              if (Math.abs(e.videoState.endedAt - endedAtRef.current) >= 5) {
-                insertPlayerEventLog({ ...e.videoState, startedAt: endedAtRef.current || e.videoState.startedAt })
-                if (e.type === 'progress') {
-                  insertProgramProgress(e.progress)
+      moment().isBefore(moment(programContent.publishedAt)) &&
+      currentUserRole !== 'app-owner' ? (
+        <StyledUnpublishedBlock>
+          <StyledIcon as={LockIcon} className="mb-3" />
+          <p>{formatMessage(ProgramContentPageMessages.ProgramContentBlock.theContentWillAt)}</p>
+          <p>
+            {`${moment(programContent.publishedAt).format('YYYY/MM/DD HH:mm')} ${formatMessage(
+              commonMessages.text.publish,
+            )}`}
+          </p>
+        </StyledUnpublishedBlock>
+      ) : (!!isEquityProgramContent && moment().isAfter(moment(programContent.publishedAt))) ||
+        currentUserRole === 'app-owner' ? (
+        programContent.contentType === 'video' ? (
+          (!isBackgroundMode || programContent.videos[0]?.data?.source === 'youtube') && (
+            <ProgramContentPlayer
+              programContentId={programContentId}
+              nextProgramContent={nextProgramContent}
+              onVideoEvent={e => {
+                if (Math.abs(e.videoState.endedAt - endedAtRef.current) >= 5) {
+                  insertPlayerEventLog({ ...e.videoState, startedAt: endedAtRef.current || e.videoState.startedAt })
+                  if (e.type === 'progress') {
+                    insertProgramProgress(e.progress)
+                  }
+                  endedAtRef.current = e.videoState.endedAt
                 }
-                endedAtRef.current = e.videoState.endedAt
-              }
-              if (e.type === 'ended') {
-                insertPlayerEventLog({ ...e.videoState, startedAt: endedAtRef.current || e.videoState.startedAt })
-                insertProgramProgress(1)?.then(() => refetchProgress())
-              }
-            }}
+                if (e.type === 'ended') {
+                  insertPlayerEventLog({ ...e.videoState, startedAt: endedAtRef.current || e.videoState.startedAt })
+                  insertProgramProgress(1)?.then(() => refetchProgress())
+                }
+              }}
+            />
+          )
+        ) : programContent.contentType === 'ebook' ? (
```

Comment:
`!!isEquityProgramContent` 
ebook also need to be checked

Reply:
Sorry, I didn't see it clearly. Perhaps this part can be simplified by extracting the condition and using a single-level ternary operator.

Reply:
Thanks for review, it has been adjusted after discussion.